tests/integration_tests/test_clients_integration.py

```diff
@@ -30,11 +30,32 @@ def test_create_pull_request_comment():
     assert res
 
 
-def test_get_comparison_to_baseline(monkeypatch: pytest.MonkeyPatch):
-    monkeypatch.setenv("CONBENCH_URL", "https://conbench.ursa.dev/")
+@pytest.mark.parametrize(
+    ["conbench_url", "commit", "expected_len", "expected_bip"],
+    [
+        (
+            "https://conbench.ursa.dev/",
+            "bc7de406564fa7b2bcb9bf055cbaba31ca0ca124",
+            8,
+            True,
+        ),
+        (
+            "https://velox-conbench.voltrondata.run",
+            "2319922d288c519baa3bffe59c0bedbcb6c827cd",
```

Comment:
I did test that this would have failed on the `main` branch but works with this PR.